src/client/cypress/e2e/testHelpers.js

```diff
@@ -144,6 +143,10 @@ export const createBorehole = values => {
     });
 };
 
+export const createBoreholeLoginAsAdminInBoreHoleEditForm = values => {
```

Comment:
PP: Alternativvorschlag für einen aussagekräftigeren Methodennamen
```suggestion
export const createAndEditBoreholeAsAdmin = values => {
```